documentation/secure_storage.md

```diff
@@ -74,16 +74,17 @@ Below is an excerpt from the specification listing the most vital requirements:
    Typically, an implementation may rely on the REE for that purpose (protection
    level 100) or on hardware assets controlled by the TEE (protection level
    1000).
+   The current implementation does *not* provide any protection against
+   rollback, although most of the code needed to do so is already implemented.
```

Comment:
> Shouldn't this state that current optee proposes level 100.

No, currently `gpd.tee.trustedStorage.antiRollback.protectionLevel` is 0, because we don't provide *any* protection against rollback. The GP Internal Core API Spec v1.1 is inconsistent on this, because section 5.2 says "The Trusted Storage MUST provide a minimum level of protection against rollback attacks" and table 5-1 does not list 0 as a valid value for protection level. On the other hand, table 4-14 lists 0, 100 and 1000 as acceptable values (0 meaning"No anti rollback protection").